jujugui/static/gui/src/app/assets/css/inspector/_juju-inspector.scss

```diff
@@ -149,38 +147,44 @@
             border-bottom: 1px solid rgba(255, 255, 255, 0.1);
         }
         button.unit-action-button,
-        button.remove-relation,
-        .controls button,
-        .inspector-buttons a,
-        .inspector-buttons button {
+        button.remove-relation {
             display: inline-block;
             vertical-align: middle;
             width: 47%;
             margin: 20px 0;
             padding: 0;
             border: none;
-            color: #505050;
+            color: $inspector-text-color;
             text-align: center;
             text-decoration: none;
             line-height: 30px;
         }
-        .controls button,
         .inspector-buttons a,
         .inspector-buttons button {
-            color: $inspector-text-color;
-
-            &.cancel-button,
-            &.cancel {
-                background:  $inspector-dark;
-            }
             &.rerender-config {
                 background-color: $confirm-button-color;
+                border: none;
             }
             &.initiate-destroy,
             &.confirm-num-units,
             &.confirm {
                 float: right;
-                background-color: $confirm-button-color;
+            }
+        }
+
+        .inspector-buttons a,
+        .inspector-buttons button,
+        button,
```

Comment:
I dont think we should target raw elements with styling. Doesnt look like there are any button elements without a button class.